### ISSUE TITLE
Multi-output support for CutMix + MixUp

### ISSUE THREAD

**Issue by LukeWood** [keras-team/keras-cv#1025]
multi-output labels are typically dictionaries with Tensor keys.  We can iterate over these and blend them.

**Comment by LukeWood** [comment]
@ebursztein will provide more info

**Comment by innat** [comment]
This would be a very useful feature for multi-output classifiaction task. This was heavily dominated in one of a kaggle competition, [link](https://www.kaggle.com/competitions/bengaliai-cv19/discussion/126504). With this kaggle-data, I've prepared a sample code to get started. 

https://colab.research.google.com/drive/1EuBv-kZxyw6PkK3SmGeHHv_gF8A_LsrA?usp=sharing

**Comment by divyashreepathihalli** [comment]
@innat thank you for the colab, have you opened a PR for this?

**Comment by sachinprasadhs** [comment]
Thanks for reporting the issue! We have consolidated the development of KerasCV into the new KerasHub package, which supports image, text, and multi-modal models. Please read https://github.com/keras-team/keras-hub/issues/1831. KerasHub will support all the core functionality of KerasCV.

KerasHub can be installed with !pip install -U keras-hub. Documentation and guides are available at [keras.io/keras_hub](https://keras.io/keras_hub/).

With our focus shifted to KerasHub, we are not planning any further development or releases in KerasCV. If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).

**Comment by innat** [comment]
@sachinprasadhs 
> If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).

Firstly, identifying which feature is required or missing can be done effectively, either by practitioners or the Keras team. Tickets in `keras-cv` were created only when such features were **NOT** available or when a specific need was encountered. Commenting after a significant delay and requesting the recreation of the same issue is neither efficient nor appropriate. Kindly move this ticket (and any other similar tickets with the same context) from `keras-cv` to `keras-hub`.

**Comment by innat** [comment]
.
